fsmonitor.c

```diff
@@ -237,12 +281,15 @@ void refresh_fsmonitor(struct index_state *istate)
 void add_fsmonitor(struct index_state *istate)
 {
 	unsigned int i;
+	int hook_version;
```

Comment:
Maybe we want to restrict the scope of this variable to the `if()` block below, i.e. move the declaration inside those curly braces?